src/anomalib/models/image/glass/lightning_model.py

```diff
@@ -0,0 +1,161 @@
+import torch
+from torch import nn
+from torch import optim
+
+from lightning.pytorch.utilities.types import STEP_OUTPUT
+
+from anomalib.data import Batch
+from anomalib.models.components import AnomalibModule
+from anomalib.models.components import AnomalibModule
+from anomalib.metrics import Evaluator
+from anomalib.post_processing import PostProcessor
+from anomalib.pre_processing import PreProcessor
+from anomalib.visualization import Visualizer
+
+from .loss import FocalLoss
+from .torch_model import GlassModel
+
+class Glass(AnomalibModule):
```

Comment:
The class is missing the following mandatory properties: `learning_type` and `trainer_arguments`. Without the implementation of these properties, it won't run. You can have a look at the other models for example implementations.